tests/v4/test_uxarray_fieldset.py

```diff
@@ -91,7 +91,6 @@ def test_fesom_channel(ds_fesom_channel, uvw_fesom_channel):
     pset.execute(endtime=timedelta(days=1), dt=timedelta(hours=1))
 
 
-@pytest.mark.skip(reason="_index_search.search_time_indices needs major refactoring")
 def test_fesom2_square_delaunay_uniform_z_coordinate_eval():
```

Comment:
Saw that this is still failing, but then saw its https://github.com/OceanParcels/Parcels/issues/2018#issue-3100586169